rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,180 @@
+like the constructs and their usage in PHP code is a user facing change. The actual C implementation and tests are
+a technical change.
+
+==== TC membership ====
+
+The TC consists of five elected members.  Any core developer is eligible to run for the TC.  One member of the TC,
+selected by the TC members themselves, is the TC Secretary.
+
+The TC Secretary is responsible for:
+
+  * Administering the election of the next set of TC members.
+  * Documenting and communicating any decisions of the TC.
+  * In case of a tie, the Secretary's vote will be considered the winning position.
+
+=== Elections ===
+
+The election process takes place annually, concurrent with the election of the Release Managers in approximately
+March/April.  (The exact date may vary year to year.)  The sitting TC Secretary officially begins the process with
+a call for nominations.  The nomination period lasts for one week.
+
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
+
+Once nominations are over, the voting process will begin.  Eligible voters are the same as for any RFC.
+
+If there are five or fewer nominees, there is no vote and all nominated individuals are elected.
+
+If there are six or more nominees, a two week vote using the Single Transferable Vote method will take place.  The five
+winning candidates at the end of the vote are elected.
+
+The newly elected TC members will elect from their own number a TC Secretary by simple majority.
+
+=== Vacancies ===
+
+Any TC member may resign at any time via an email to the Internals mailing list.  If the resigning member is the TC
+Secretary, then the TC will elect a new Secretary from its number.
+
+Should a TC member become inactive for more than 30 days, the TC may declare that member's seat vacant, by a majority vote.
+If the vote passes, the TC member is removed.
+
+If the number of TC members is reduced to three or fewer and there are more than two months until the next regularly
+scheduled election, the Secretary must call for a special election for the vacant seats.  The election process is the
+same as for any other election, but the newly elected TC members will serve only until the next regular election.
+
+Should the number of TC members fall to zero, the senior-most Release Manager of the latest stable release will step
+ in to act as TC Secretary and immediately call for a new vacancy election.
+
+==== Work flow ====
+
+The TC is primarily a reactive body, and is not expected to proactively search for issues.  However, TC members
+are permitted to comment on any open issue, and contributors are encouraged to give their comments appropriate weight.
```

Comment:
Are all comments made by TC members considered to be in their capacity as a TC member? Or do they need to declare that they are commenting in their role as a TC member?

I'm asking, because how will contributors know that a person is a member of the TC, particularly if they are new to the project, and thus know they should weight these comments more?

Reply:
I think we should actually rephrase  / remove this as the comments on behalf of TC should be done only after discussion with other members. Thinking about that, it should not be done if not requested as there is no point for the TC members to discuss issues where there is no conflict.

Reply:
I removed this part.